lib/statistics.js

```diff
@@ -1,43 +1,33 @@
 'use strict';
 
-const Stats = require('fast-stats').Stats;
-
-function percentileName(percentile) {
-  if (percentile === 0) {
-    return 'min';
-  } else if (percentile === 100) {
-    return 'max';
-  } else if (percentile === 50) {
-    return 'median';
-  }
-  return `p${String(percentile).replace('.', '_')}`;
-}
-
 class Statistics {
   constructor() {
-    this.stats = new Stats();
+    this.values = [];
   }
 
   add(value) {
-    this.stats.push(value);
+    this.values.push(value);
     return this;
   }
 
-  summarize(options) {
-    if (isNaN(this.stats.amean())) {
+  summarize() {
+    const values = this.values;
```

Comment:
To not require es6 support in the browser for the browserified version, maybe use this.values.slice(). As a general note, the node module requires node 6, with support for es6 syntax. Maybe babelify is required to transcode to es5 in browserify. Otherwise I see potential issues with using api in the module that's not supported by browsers. We won't catch that in any of the tests.